release.nix

```diff
@@ -149,36 +149,32 @@ let
       migration-tests = jobs.x86_64-w64-mingw32.migration-tests.x86_64-linux;
     };
 
-    # Fully-static linux binary (placeholder - does not build)
-    cardano-wallet-jormungandr-linux64 = let
-      name = "cardano-wallet-jormungandr-${project.version}";
-      tarname = "${name}-linux64.tar.gz";
-    in pkgs.runCommand "${name}-linux64" {
-      buildInputs = with pkgs.buildPackages; [ gnutar gzip binutils ];
-    } ''
-      cp -R ${jobs.musl64.cardano-wallet-jormungandr.x86_64-linux}/bin ${name}
-      chmod -R 755 ${name}
-      strip ${name}/*
-
-      mkdir -p $out/nix-support
-      tar -czf $out/${tarname} ${name}
-      echo "file binary-dist $out/${tarname}" > $out/nix-support/hydra-build-products
-    '';
+    # Fully-static linux binaries
+    cardano-wallet-jormungandr-linux64 = import ./nix/linux-release.nix {
+      inherit pkgs;
+      exes = [ jobs.musl64.cardano-wallet-jormungandr.x86_64-linux ];
+    };
+    cardano-wallet-byron-linux64 = import ./nix/linux-release.nix {
+      inherit pkgs;
+      exes = [
+        jobs.musl64.cardano-wallet-byron.x86_64-linux
+        # SRE-83 dependencies fail to build
+        # jobs.musl64.cardano-node.x86_64-linux
+      ];
+    };
 
     # macOS binary and dependencies in tarball
-    cardano-wallet-jormungandr-macos64 = let
-      name = "cardano-wallet-jormungandr-${project.version}";
-      tarname = "${name}-macos64.tar.gz";
-    in pkgs.runCommand "${name}-macos64" {
-      buildInputs = with pkgs.buildPackages; [ gnutar gzip binutils nix ];
-    } ''
-      cp -R ${jobs.native.cardano-wallet-jormungandr.x86_64-darwin}/bin ${name}
-      chmod -R 755 ${name}
-
-      mkdir -p $out/nix-support
-      tar -czf $out/${tarname} ${name}
-      echo "file binary-dist $out/${tarname}" > $out/nix-support/hydra-build-products
-    '';
+    cardano-wallet-jormungandr-macos64 = import ./nix/macos-release.nix {
+      inherit pkgs;
+      exes = [ jobs.native.cardano-wallet-jormungandr.x86_64-darwin ];
+    };
+    cardano-wallet-byron-macos64 = import ./nix/macos-release.nix {
+      inherit pkgs;
+      exes = [
+        jobs.native.cardano-wallet-byron.x86_64-darwin
+        jobs.native.cardano-node.x86_64-darwin
```

Comment:
Why bundle `cardano-node` with the wallet here? I don't think our release assets should contain anything else than the wallet and the DLLs / runtime dependencies needed to run it.

Reply:
To save users the not-insignificant hassle of finding the correct version of cardano-node and downloading then installing the macos build for it. To save ourselves from support requests where users are running the wrong version of cardano-node. Because cardano-node cannot yet report its own version. And I don't think there are even macos binaries of cardano-node to available to download.
We can remove it sure, but it costs so little to include in the tarball.

Reply:
> And I don't think there are even macos binaries of cardano-node to available to download.

:man_facepalming: ... you have a point. There are actually not even Linux assets at the moment :upside_down_face: ... 
And since it is not possible to (easily) know the version of the `cardano-node` executable at the moment .... 

Fair enough!